### PR TITLE
app-editor/{vscodium|vscode}: disable update server on install

### PR DESCRIPTION
**Pull request by joecool1029**
This is also applicable to vscode. As portage manages
the updates for these packages it is undesirable to receive
notifications about updates that may not apply.

This removes the reference to the update server.

Signed-off-by: Joe Kappus <joe@wt.gd>

### CODE REVIEW

File: app-editors/vscodium/vscodium-1.70.1.ebuild [hunk 1]
```diff
@@ -88,6 +88,9 @@ src_install() {
 	# Cleanup
 	rm "${S}/resources/app/LICENSE.txt" || die
 
+	# Disable update server
+        sed -i "/updateUrl/d" ${S}/resources/app/product.json || die
```
Comment (arthurzam) [inline app-editors/vscodium/vscodium-1.70.1.ebuild:92]: looks like indentation broke :)

**Review by arthurzam — CHANGES_REQUESTED** [review]
Can you also do the same to `vscode` in same PR?
Also, this needs a revision bump, as it affects installed files.

**Comment by joecool1029** [comment]
> Can you also do the same to `vscode` in same PR? Also, this needs a revision bump, as it affects installed files.

Is it ok to remove the old revisions? Or do I leave those in?

**Review by arthurzam — CHANGES_REQUESTED** [review]
Please fix indentation of you changes!
It should be a single TAB in all of them 😃

**Comment by joecool1029** [comment]
> Please fix indentation of you changes!
> It should be a single TAB in all of them 😃 

Sorry about that, I will fix the editor situation and try to get more sleep!

**Comment by joecool1029** [comment]
https://github.com/VSCodium/vscodium/discussions/1189 Some discussion with upstream on the update check situation.

**Review by arthurzam — APPROVED** [review]

**Pull request closed without merging**